google/pubsub/v1/pubsub.proto

```diff
@@ -30,6 +30,15 @@ option java_outer_classname = "PubsubProto";
 option java_package = "com.google.pubsub.v1";
 option php_namespace = "Google\\Cloud\\PubSub\\V1";
 
+option (google.api.metadata) = {
+  product_name: "PubSub"
+  product_uri: "https://cloud.google.com/pubsub/"
+};
+
+option (google.api.resource_definition) = {
```

Comment:
Project does need to be a common resource, not defined here.

Reply:
Originally this resource_definition was going to be defined in `google/api/{something}.proto` so that if you imported the annotations the resource would implicitly be defined. IIUC this makes it such that generated class for the project resource will be common amongst those who use it.

Reply:
Also, to be clear, I agree that this should be defined commonly elsewhere.

Reply:
@lukesneeringer The Project message was removed in https://github.com/googleapis/api-common-protos/pull/20/files. Are there plans to reintroduce it, or are we not defining a common resource (there)?